<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>431</width>
      <height>119</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="AllocateFileLoad">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Allocate FileLoad</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeAllocateFileLoad">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>DeAllocate FileLoad</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_1">
       <item>
        <widget class="QCheckBox" name="AutoplayQtMultiMedia">
         <property name="text">
          <string>Autoplay</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AllocateQtMultimedia">
         <property name="text">
          <string>Allocate QtMultimedia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="RepeatsQtMultimedia">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeAllocateQtMultimedia">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>DeAllocate QtMultimedia</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="AutoplayQtAV">
         <property name="text">
          <string>Autoplay</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AllocateQtAV">
         <property name="text">
          <string>Allocate QtAv</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="RepeatsQtAV">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeAllocateQtAV">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>DeAllocate QtAv</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="OutputMemoryUse">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MaxMemoryUsed">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="video_widget_layout"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
